mkdud

```diff
@@ -605,6 +612,17 @@ sub file_type
       exit 1;
     }
 
+    # check if rpm contains a driver update
+    # (a directory /linux/suse/foo-bar or /<NUMBER>/linux/suse/foo-bar exists)
+    for my $l (`rpm --nosignature -qp -lv $_[0] 2>$tmp_err`) {
```

Comment:
Not only are one-letter variable names not the best choice, but in this instance I initially confused it with $1 and wondered what $1 might be in this context in Perl (I am sure $1 could be used somehow here, too).

I would have chosen $lines or $rpms for this one and @words for the other one; this would make this very much clearer.